conf/uppmax.config

```diff
@@ -107,6 +106,7 @@ process {
 
 params {
   clusterOptions = false
+  params.rlocation = "$HOME/R/nxtflow_libs/"
```

Comment:
This is already inside the `params` scope, so you've essentially defined it as `params.params.rlocation`.

```groovy
 params {
    clusterOptions = false
    rlocation = "$HOME/R/nxtflow_libs/"
```